server/utils/constants.js

```diff
@@ -168,6 +168,8 @@ const RedisKeys = {
   UPLOAD_PHOTOS_ERROR: 'upload-photos.error',
   USED_CHECKER: 'used-checker',
   WHAT_TYPE_OF_ITEM_IS_IT: 'what-type-of-item-is-it',
+  WHERE_IS_ITEM: 'where-is-item',
```

Comment:
I think the RedisKey 'WHERE_IS_ITEM' was removed in a previous PR. Has it snuck back in during a merge or something?

Reply:
Fixed